### DISCUSSION TITLE
Skip Jobs with the same Job Paramters

### DISCUSSION THREAD

**Discussion by goafabric** [jobrunr/jobrunr#842] [General]
I have another rather simple question, based on what spring batch is doing.
With spring batch, jobs with the same Job Parameters just don't get run twice.

With JobRunr and JobRequest i was not able to achieve this.
I understand that Job Paramters here, are just a simple Property attached to the JobRequest.

But nontheless, would it be possible to skip jobs with the same Job Parameters applied ?

**Comment by rdehuyss** [comment]
See https://github.com/jobrunr/jobrunr/issues/236.
